wsgi/odyssy/events/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import get_object_or_404
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import render
+from .models import Event
+
+
+def event(request):
+    events_list = Event.objects.order_by('-event_date')
+    paginator = Paginator(events_list, 3)
+    page = request.GET.get('page')
```

Comment:
There is no such method in `Paginator` class which accepts a default page value. I guess we will have to stick with this implementation.